pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-details-directive";
+  version = "0.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "eL1qZ/eGohhoq/DmpZczQNfnpv1xsYkN6chW+Sh3s4s=";
+  };
+
+  propagatedBuildInputs = [
+    sphinx
+  ];
+
```

Comment:
```suggestion

  nativeCheckInputs = [
    pytestCheckHook
  ];

```

Reply:
```
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-7.2.1, pluggy-1.0.0
rootdir: /build/sphinxcontrib-details-directive-0.1.0, configfile: setup.cfg
collected 0 items                                                              

============================ no tests ran in 0.01s =============================
/nix/store/blpvf60m29q02c0lc5fyhim30ma4y1vv-stdenv-linux/setup: line 1596: pop_var_context: head of shell_variables not a function context
```
I'm not sure what to do with this and it fails to run.

Reply:
I think the tests are not in the sdist and we would need to fetch from github to run them.